src/sth_configuration.js

```diff
@@ -0,0 +1,38 @@
+/* globals module, process */
+
+(function() {
+  "use strict";
+
+  var ENV = process.env;
```

Comment:
Is this `process.env` a "list" or arguments passed to the application? Or is somehow a pointer to any configuration file?